packages/orderbook/test/js_api/gui.test.ts

```
@@ -576,31 +576,30 @@ describe('Rain Orderbook JS API Package Bindgen Tests - Gui', async function ()
 		it('should throw error if deposit token is not found in gui config', () => {
-			expect(() => gui.saveDeposit('token3', '1')).toThrow(
-				'Deposit token not found in gui config: token3'
-			);
+			const result = gui.getDepositPresets('token3');
+			expect(result.error.msg).toBe('Deposit token not found in gui config: token3');
 		});
 
 		it('should remove deposit', async () => {
 			gui.saveDeposit('token1', '50.6');
-			let deposits: TokenDeposit[] = gui.getDeposits();
+			let deposits = extractWasmEncodedData<TokenDeposit[]>(gui.getDeposits());
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'deposits' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)